app/models/subject_queue.rb

```diff
@@ -79,6 +79,10 @@ def below_minimum?
   end
 
   def next_subjects(limit=10)
-    set_member_subject_ids[0..limit-1]
+    if user_id
+      set_member_subject_ids[0..limit-1]
```

Comment:
Good point about talk, i didn't think about that. Having more "voices" on subject threads in a time period is nice. Leave it as is and we'll see how it goes.